pkg/commands/loaders/branches.go

```diff
@@ -27,24 +27,30 @@ type BranchLoaderConfigCommands interface {
 	Branches() (map[string]*config.Branch, error)
 }
 
+type BranchInfo struct {
```

Comment:
define in loaders instead of git_commands to avoid import cycles.

Reply:
It sucks that we have to do this: we ought to either merge the loaders directory into the git_commands directory if it's causing these kinds of issues. But we can do that in another PR

Reply:
I've created an issue for this: https://github.com/jesseduffield/lazygit/issues/2248